Dash oil gas ternary (#99)
plotly/dash-sample-apps: apps/dash-oil-gas-ternary/README.md

## dash-oil-gas-ternary

`dash-oil-gas-ternary` creates a dashboard for mineral composition evaluations from natural gas wells.  
By pairing a geographic map of well locations with ternary diagram, this app helps geologists to find out results of gas wells with relationship to formations or rock types.
This is a demo of Dash interactive Python framework developed by [Plotly](https//plot.ly/).

![Animated](assets/Screencast.gif)

## Screenshots
![initial](assets/Screenshot.png)

## Requirements
We suggest you to create a separate virtual environment running Python 3 for this app, and install all of the required dependencies there. Run in Terminal/Command Prompt:

```

> git clone https://github.com/plotly/dash-oil-gas-ternary.git

Reviewer: @shammamah told me these repo will be archived so probably put the monorepo link here and have them cd into dash-sample-apps/apps/dash-oil-gas-ternary

Reviewer 2: I'd add a migration/deprecated notice in original repo readme.

> ## Dash-oil-gas-ternary

Reviewer 3: ```suggestion
## dash-oil-gas-ternary
```

`dash-oil-gas-ternary` creates a dashboard for mineral composition evaluations from natural gas wells.  

> By pairing geographic map of well locations with ternary diagram, this app helps geologists to find out results of gas wells with relationship to formations or rock types.

Reviewer 3: ```suggestion
By pairing a geographic map of well locations with a ternary diagram, this app helps geologists visualize gas wells and the constituent minerals of the earth that surrounds them.
```

This is a demo of Dash interactive Python framework developed by [Plotly](https//plot.ly/).

![Animated](assets/Screencast.gif)

## Screenshots
![initial](assets/Screenshot.png)

## Requirements
We suggest you to create a separate virtual environment running Python 3 for this app, and install all of the required dependencies there. Run in Terminal/Command Prompt:

```
git clone https://github.com/plotly/dash-oil-gas-ternary.git
cd dash-oil-gas-ternary
python3 -m virtualenv venv
```
In UNIX system: 

```
source venv/bin/activate
```
In Windows: 

```
venv\Scripts\activate
```

To install all of the required packages to this environment, simply run:

```
pip install -r requirements.txt
```

and all of the required `pip` packages, will be installed, and the app will be able to run.

## How to use the app
Run this app locally by:
```
python app.py
```
Open http://127.0.0.1:8050/ in your browser.

> Select data points from well map, ternary map or bar graph to visualize cross-filtering to other plots.

Reviewer 3: ```suggestion
Select a subset of data points from the well map, ternary map or bar graph to visualize cross-filtering to other plots.
```

> Selection could be done by clicking on individual data points or using the lasso tool to capture multiple data points or bars. Hold SHIFT

Reviewer 3: ```suggestion
Selection can be done by clicking on individual data points, or by using the lasso tool to capture multiple data points or bars. Hold the SHIFT
```

> button while click-and-drag for multi-region selection.

Reviewer 3: ```suggestion
key while clicking and dragging for multi-region selection.
```
